pyexchange/korbit.py

```diff
@@ -0,0 +1,361 @@
+# This file is part of Maker Keeper Framework.
+#
+# Copyright (C) 2020 MikeHathaway
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+from pprint import pformat
+from pyexchange.api import PyexAPI
+import hmac
```

Comment:
`hmac`, `hashlib`, `base64', and `dateutil.parser` seem to be unused imports.

Reply:
Removed unused imports